libminifi/src/c2/C2Agent.cpp

```diff
@@ -342,6 +342,31 @@ void C2Agent::serializeMetrics(C2Payload &metric_payload, const std::string &nam
   }
 }
 
+/*
+void C2Agent::serializeMetrics(C2Payload &metric_payload, const std::string &name, const std::vector<state::response::SerializedResponseNode> &metrics, bool is_container, bool is_collapsible) {
```

Comment:
What about this, why comment?

Reply:
Committed that accidentally. Removed the comments in latest commit.